resolver/resolver.cc

```diff
@@ -2005,6 +2012,23 @@ class ResolveTypeMembersAndFieldsWalk {
             }
 
             if (symbol.isClassOrModule()) {
+                // This is the same as the implementation of T::Generic.[] in calls.cc
```

Comment:
Non-blocking, but it would be nice if this list was in one and only one place.

Reply:
Yep, I agree. There are a few different places that this sort of check is done.